packages/client/hmi-client/src/App.vue

```diff
@@ -52,6 +55,7 @@ watch(
 		} else {
 			const id = projectId as string;
 			project.value = await ProjectService.get(id);
+			resources.setActiveProject(project.value);
```

Comment:
I do not want to set the active project to null when the route param does not include projectId because it conflicts with the case of viewing a specific document as one of project assets. In such a case, the route parameter does not contain a valid project-id which will invalidate the active project, which would be wrong!

Reply:
... Maybe we should not save assets under a project within the store.